### ISSUE TITLE
For def _prob_in_top_k

### ISSUE THREAD

**Issue by Brankozz** [davidmrau/mixture-of-experts#31]
Hey, sorry for bothering! I am just confused that you wrote the function as:
prob_if_in = normal.cdf((clean_values - threshold_if_in)/noise_stddev)
prob_if_out = normal.cdf((clean_values - threshold_if_out)/noise_stddev)

In my perspective, in the origin paper, regardless of threshold_if_in or threshold_if_out, it should be the initial logits(after adding the noise) before the softmax(initial logits). After I run the code, I find out the inputted threshold_if_in/out are actually from the softmax(initial logits).

Also in this commond  "is_in = torch.gt(noisy_values, threshold_if_in)" the noisy_values are the initial logits, the threshold_if_in is the output of softmax(initial logits)

**Comment by tulilin** [comment]
@davidmrau I also found this problem.
